src/pluginload.h

```diff
@@ -53,6 +53,9 @@ total_loading_plugins (void);
 int
 nasl_plugin_add (char *, char *);
 
+int
+check_notus_vt_list (char *, char *);
```

Comment:
Same as above. I would not use the name notus. Why not something like check_csv_vt_list ?
Or even just check_csv_file.